util/info.c

```diff
@@ -228,29 +229,53 @@ static const char *param_type(enum fi_param_type type)
 	}
 }
 
+static char * get_var_prefix(const char *prov_name)
+{
+	int i;
+	char *prefix;
+
+	if (!prov_name) {
+		return NULL;
+	} else {
+		if (asprintf(&prefix, "FI_%s", prov_name) < 0)
+			return NULL;
+		for (i = 0; i < prefix[i]; ++i)
```

Comment:
Did you mean to use strlen(prefix) here?

Reply:
@shefty yes, I can replace this with strlen(prefix) for better code style

Reply:
I don't think this is a coding style issue.  It looks more like a correctness issue.  I'm not sure what the check i < prefix[i] is intended to do.